File: Bungii.Test.Integration/src/test/java/com/bungii/android/pages/menus/SupportPage.java
package com.bungii.android.pages.menus;

import com.bungii.common.core.PageBase;
import org.openqa.selenium.WebElement;

public class SupportPage extends PageBase {

    public WebElement Header_SupportPage() { return findElement("//android.widget.TextView[@text='SUPPORT']", LocatorType.XPath); }

    public WebElement Title() { return findElement("com.bungii.customer:id/feedback_text_view_title", LocatorType.Id); }

    public WebElement TextField() { return findElement("com.bungii.customer:id/feedback_edit_text", LocatorType.Id); }

    public WebElement Button_Send() { return findElement("com.bungii.customer:id/feedback_send_button", LocatorType.Id); }

    public WebElement Error_Blank() { return findElement("com.bungii.customer:id/textinput_error", LocatorType.Id); }

}

File: BungiiAutomation/Bungii.Test.Regression.Android.Integration/Pages/MenuPages/SupportPage.cs
using OpenQA.Selenium;
using OpenQA.Selenium.Support.PageObjects;

namespace Bungii.Test.Regression.Android.Integration.Pages
{
    class SupportPage
    {
        public SupportPage(IWebDriver driver)
        {
           PageFactory.InitElements(driver, this);
        }

        [FindsBy(How = How.XPath, Using = "//android.widget.TextView[@text='SUPPORT']")]
        public IWebElement Header_SupportPage { get; set; }

        [FindsBy(How = How.Id, Using = "com.bungii.customer:id/feedback_text_view_title")]
        public IWebElement Title { get; set; }

        [FindsBy(How = How.Id, Using = "com.bungii.customer:id/feedback_edit_text")]
        public IWebElement TextField { get; set; }

        [FindsBy(How = How.Id, Using = "com.bungii.customer:id/feedback_send_button")]
        public IWebElement Button_Send { get; set; }

        [FindsBy(How = How.Id, Using = "com.bungii.customer:id/textinput_error")]
        public IWebElement Error_Blank { get; set; }
    }
}
